on-boot-script/README.md

```diff
@@ -3,7 +3,7 @@
 ## Features
 
 1. Allows you to run a shell script at S95 anytime your UDM starts / reboots
-1. Persists through reboot and **firmware updates**! It is able to do this because Ubiquiti caches all debian package installs on the UDM in /mnt/data, then re-installs them on every boot
+1. Persists through reboot and **firmware updates**! It is able to do this because Ubiquiti caches all debian package installs on the UDM in /mnt/data, then re-installs them on reset of unifi-os container.
```

Comment:
Interesting point here! So a `unifi-os restart` of said container actually caches and re-installs all used Debian packages?

Reply:
Restart would not. There’s a hidden “unifi-os reset” command which recreates the container.

Reply:
The `deb` package will remain installed as long as the unifi-os container is not recreated. Either a firmware upgrade or `unifi-os` would cause the container to be recreated.

Reply:
Ah, thanks for the reminder and clarification. I forgot about the reset command.